clip_image.py

```diff
@@ -77,13 +76,13 @@ def _create_embeddings(self, document_batches_generator: Iterable):
                 if self.use_default_preprocessing:
                     images = [
                         Image.fromarray(
-                            np.moveaxis(np.uint8(blob*255), 0, -1)
+                            np.uint8(blob * 255)
                         ) for blob in blob_batch
                     ]
                     tensors = [self.preprocess(img) for img in images]
                     tensor = torch.stack(tensors)
                 else:
-                    tensor = torch.from_numpy(np.array(blob_batch))
+                    tensor = torch.from_numpy(np.array([np.moveaxis(b, -1, 0) for b in blob_batch]))
```

Comment:
why are u changing here anything? if in the README u say that the blob must be of certain shape?

Reply:
yes the input shape is (h, w, 3), the model requires the input (3, h, w).

Reply:
preprocessing function is doing that for us